### PR TITLE
Updated readme file with recent changes done to the framework

### PR DESCRIPTION
**Pull request by rohandudam**
- Updated setup section with our wiki links
- Updated test run command section. Made it more readable and provided commands to run along with third party integrations.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -32,154 +32,175 @@ We've implemented some version of this framework at several [clients](https://qx
...
+	`python -m pytest tests/test_mobile_bitcoin_price --mobile_os_version <android version> --device_name <simulator> --app_path <.apk location on local> --remote_flag Y`  
+	**Note:** For running tests on Browserstack/Sauce Labs, update the Browser_Plaform, Username and AccessKey in `.env.remote` from your Browserstack/Sauce Labs account. Refer our wiki page for more details: [Integrate our Python Selenium automation framework with Cloud Services ](https://github.com/qxf2/qxf2-page-object-model/wiki/Integration-with-Cloud-Services)
+
+- **Run Test along with Tesults**  
+	`python -m pytest tests/test_example_form.py --tesults Y`  
+	**Note:** For running the test along with Tesults, update the .env file with target_token and run the above command. Refer env_conf file for configuration.
+		
+- **Run Tests along with ReportPortal**  
+	`python -m pytest -k example --reportportal`  
+	**Note:** For running the test along with ReportPortal, update the .env file with report portal credential details and run the above command. Refer our wiki page for more details: [Integration with ReportPortal](https://github.com/qxf2/qxf2-page-object-model/wiki/Integration-with-ReportPortal)
```
Comment (sravantit25) [inline Readme.md:191]: In this line "update the .env file with report portal credential details " Can you make it ReportPortal.

In the corresponding wiki, can you rephrase slightly the first paragraph.
"Report Portal services are mainly used for analyzing test results. We integrate report analysis into our framework, which collects and stores test log information and screenshot images in the Report Portal."

In general, in the wiki, wherever the word reportportal is referred can you make it "ReportPorta" for consistency.

File: Readme.md [hunk 2]
```diff
@@ -32,154 +32,175 @@ We've implemented some version of this framework at several [clients](https://qx
...
+	`python -m pytest tests/test_example_form.py --tesults Y`  
+	**Note:** For running the test along with Tesults, update the .env file with target_token and run the above command. Refer env_conf file for configuration.
+		
+- **Run Tests along with ReportPortal**  
+	`python -m pytest -k example --reportportal`  
+	**Note:** For running the test along with ReportPortal, update the .env file with report portal credential details and run the above command. Refer our wiki page for more details: [Integration with ReportPortal](https://github.com/qxf2/qxf2-page-object-model/wiki/Integration-with-ReportPortal) 
+
+- **Run Tests along with TestRail**  
+	`python -m pytest -k example --testrail_flag Y --test_run_id <testrail run id>`  
+	**Note:** For running the test along with TestRail, update the .env file with TestRails credential details and run the above command. Refer our wiki page for more details: [Integrate our Python test automation framework with Testrail](https://github.com/qxf2/qxf2-page-object-model/wiki/Integration-with-TestRail-using-Python)
```
Comment (sravantit25) [inline Readme.md:195]: There is a typo " update the .env file with TestRails credential", should be "TestRail"

**Review by sravantit25 — APPROVED** [review]
@rohandudam The updated Readme file is really better and easy to navigate through.
I have a few small observations mostly related to rephrase, typo corrections. 

Looks good otherwise.

File: Readme.md (hunk 2 above)
Comment (rohandudam) [reply Readme.md:195]: Done

File: Readme.md [hunk 3]
```diff
@@ -32,154 +32,175 @@ We've implemented some version of this framework at several [clients](https://qx
...
+  *Runs the tests against multiple browsers.*  
+  **Example:**  
+  `python -m pytest --browser all`  
+  This will run each test against the list of browsers specified in the `conf/browser_os_name_conf.py` file (e.g., Firefox and Chrome).
+
+- **`--ver / --os_name / --os_version`**  
+  *Runs against different browser versions/OS and OS versions.*  
+  **Example:**  
+  `python -m pytest --ver 120 --os_name windows --os_version 11 --remote_flag y`  
+  This will run each test with defalut browser (chrome) and provided combination (browser version 120 windows 11)
```
[the remark this answers is not in the document]
Comment (rohandudam) [reply Readme.md:124]: Done.

File: Readme.md [hunk 4]
```diff
@@ -32,154 +32,175 @@ We've implemented some version of this framework at several [clients](https://qx
...
-	|__utils: All utility modules (email_util,TestRail, BrowserStack, Base Logger, post_test_reports_to_slack) are kept in this folder
+   |_ tests: Put your tests here
 
+   |_ utils: All utility modules (ssh_util, compare csv, compare images, Base Logger, etc) are kept in this folder
+
+   |_ conftest.py: Configuration file to add different fixtures used in py.test
+
+   |_ .env and .env.remote: For credential details. Refer env_conf and env_remote file and rename it to .env and .env_conf. 
+```
+for more details about the structure, refer our wiki page [here](https://github.com/qxf2/qxf2-page-object-model/wiki/Repository-details)
```
[the remark this answers is not in the document]
Comment (rohandudam) [reply Readme.md:93]: Done.

File: Readme.md [hunk 5]
```diff
@@ -32,154 +32,175 @@ We've implemented some version of this framework at several [clients](https://qx
...
-	|__conf: For all configurations 	
+   |_ conf: For all configurations files
+   
+   |_ core_helpers: Contains our web app and mobile app helpers and DriverFactory  
 
-	|__log: Log files for all tests
+   |_ endpoints: Contains our Base Mechanize, different End Points, API Player, API Interface
 
-	|__page_objects: Contains our Base Page, different Page Objects, DriverFactory, PageFactory
+   |_ integrations: Contains cross-browsers, reporting tools and reporting channel integrations (BrowserStack, SauceLabs, Lambdatest, TestRail, Tesults, Slack, Gmail)
```
[the remark this answers is not in the document]
Comment (rohandudam) [reply Readme.md:77]: Done.

File: Readme.md [hunk 6]
```diff
@@ -32,154 +32,175 @@ We've implemented some version of this framework at several [clients](https://qx
 
 Looking for ways to automate your __UI__ and __API__ tests quickly and effectively? You've come to the right place. By harnessing __AI__ and __code auto-generation__ capabilities, we've developed solutions that significantly decrease the time needed to create a fully functional test suite. For further information, please refer to the following links.
 * [Qxf2's Gen AI test automation service](https://qxf2.com/qait)
-* [Qxf2's API test automation service](https://qxf2.com/api-tests-autogenerate.html)
+* [Qxf2's API test automation service](https://qxf2.com/api-tests-autogenerate.html)
```
[the remark this answers is not in the document]
Comment (rohandudam) [reply Readme.md:35]: Looks like Github markdown does not support HTML target attribute which is required to open link in new tab. https://stackoverflow.com/questions/41915571/open-link-in-new-tab-with-github-markdown-using-target-blank

Ignoring as per our discussion.
